apps/ledger-live-desktop/src/renderer/screens/exchange/index.tsx

```diff
@@ -33,6 +38,19 @@ const LiveAppExchange = ({ appId }: { appId: string }) => {
   const manifest = localManifest || mockManifest || remoteManifest;
   const themeType = useTheme().colors.palette.type;
 
+  // Use wallet-api ids if it is using v2 multibuy manifest
+  if (urlParams && manifest?.id === DEFAULT_MULTIBUY_APP_ID) {
```

Comment:
This would not work : we do the switch between wallet api and platform sdk inside the manifest, changing its content but keeping the same id.
ex : if we want to rollback to platform sdk, we will modify the manifest that has this same id, to point to old url and use platform sdk.

You should use another manifest field, I suppose the api version : https://github.com/LedgerHQ/manifest-api/blob/main/manifests/multibuy-v2/manifest.json#L10